lib/Cws/MailBounceHandler/Handler.php

```diff
@@ -911,7 +911,7 @@ private static function parseHeader($arHeader)
 
         if (isset($arHeader['Content-type'])) {
             $ar_mr = explode(';', $arHeader['Content-type']);
-            $arHeader['Content-type'] = '';
+            $arHeader['Content-type'] = [];
```

Comment:
Hi,

Thanks for this PR but can you make a new commit by replacing `[]` with `array()` for backward compatibility